msteams-platform/build-your-first-app/build-and-run.md

```diff
@@ -35,7 +31,7 @@ Once the toolkit configures your project, you have the components to build a bas
 
 The toolkit automatically creates scaffolding for you in the `src` directory based on the capabilities you added during setup.
 
-If you create a tab during setup, for example, the `App.js` file in the `src/components` directory is important because it handles the initialization and routing of your app. It calls the [Microsoft Teams SDK](../tabs/how-to/using-teams-client-sdk.md) to establish communication between your app and Teams.
+If you create a tab during setup, for example, the `App.js` file in the `src/components` directory is important because it handles the initialization and routing of your app. It calls the [Microsoft Teams JavaScript client SDK](../tabs/how-to/using-teams-client-sdk.md) to establish communication between your app and Teams.
```

Comment:
@KirtiPereira Thanks for the feedback! I still prefer to keep the original edit because I think it does a better job establishing the example is about tabs. If we put this at the end of the sentence, someone may not read that far and think `App.js` is included in every app project when it is not.

Reply:
Ok. Thank you for the explanation. We will keep your edit.